src/resources/Catalogs/ProductListingConfigurationInterfaces.ts

```diff
@@ -0,0 +1,18 @@
+import {QueryFilterModel} from './ProductListingInterfaces';
```

Comment:
I'm sorry I don't know what that means

Reply:
If you check the other interface files, such as https://github.com/coveo/platform-client/blob/master/src/resources/Search/SearchInterfaces.ts#L52-L1225, you can see that we try to document our interfaces. This is especially important for interfaces being used as 'input/parameters', because proper documentation will offer good intellisense to the user of the library

Reply:
Nvm I know what you mean, on it.

Reply:
Makes a lot of sense. I added some documentations. Let me know if that's enough.